src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerReleaseAndTransferService.java

```diff
@@ -406,15 +483,40 @@ private void createOutMovement(final OffenderBooking booking,
             .build());
     }
 
+    private void createOutMovement(final OffenderBooking booking,
```

Comment:
I'm surprised this doesn't already exist.

Reply:
> I'm surprised this doesn't already exist.

It is a new version that takes the schedule object and populates the movement details from that